tools/stress-cluster/cluster/azure/parameters/dev.json

```diff
@@ -6,7 +6,7 @@
       "value": // add me
     },
     "groupSuffix": {
-      "value": // add me
+      "value": // add me, e.g. <your alias>
```

Comment:
It will fail because the json is invalid, which will force them to add it. Are you thinking adding some sort of log message to instruct people to fill these values in would be helpful when `ConvertFrom-Json` fails?

Reply:
Yes I was thinking this might be a something folks miss and it would be nice if there was a nice error message telling how to fix it.